lidar_base.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <stdio.h>
+
+#include "ros/ros.h"
+#include "sensor_msgs/LaserScan.h"
+#include "geometry_msgs/Twist.h"
+
+float min_range,min_range_angle;
+
+ros::Publisher vel_pub;
+geometry_msgs::Twist cmdvel;
+void chatterCallback(const sensor_msgs::LaserScan::ConstPtr& msg)
```

Comment:
It would be nice if this callback had a more descriptive name. As it stands, this could be anything.

Also, please include function comments in future so that other group members can orient more easily on the purpose of this callback.